src/share-extension/ShareExtension.tsx

```diff
@@ -27,13 +29,22 @@ export default class FelfeleShareExtension extends React.Component<{}, ShareStat
 
     public render() {
         if (this.state.store == null) {
+            Debug.log("won't render");
             return null;
         }
+        Debug.log('will render');
         return (
             <TopLevelErrorBoundary>
                 <Provider store={this.state.store!}>
                     <PersistGate loading={null} persistor={this.state.persistor!}>
-                        <SharePostEditorContainer text={this.state.value}/>
+                        <SharePostEditorContainer
+                            text={this.state.value}
+                            goBack={() => {
```

Comment:
If I understand correctly, using `goBack` also means if you cancel posting anything from the ShareExtension and that results also in an update in the store.

Unfortunately the app start time is quite long so it would be good to minimize when it happens.